### ISSUE TITLE
DATASTORE not reading XLSX files? 

### ISSUE THREAD

**Issue by prustar** [OpenDevelopmentMekong/ckanext-odm_dataset#328]
@acorbi I came across a few records that aren't displaying in CKAN resources section. There appears to be nothing wrong and the log displays nothing wrong either but yet it's not showing. I checked the file and it's plain text no formatting? 

![image](https://cloud.githubusercontent.com/assets/14996905/16337047/c009ce2e-3a12-11e6-993e-3afaf3165027.png)

![image](https://cloud.githubusercontent.com/assets/14996905/16337056/d28a709e-3a12-11e6-8903-c5f04a00e980.png)

**Comment by acorbi** [comment]
@prustar I have checked the presence of the file on the datastore and could access it successfully: https://data.opendevelopmentmekong.net/api/3/action/datastore_search?resource_id=11b68a15-e3e2-402b-b794-51fb787d6e0d

Strangely, the data is not being rendered on the table view. I even added the same data as CSV with the same result. 

If this becomes an extended issue we should look at that table rendered, by the moment I would not put much attention since that part of the frontend is not going to be available for the end user and data staff can use other tools of choice for doing it.

**Comment by prustar** [comment]
@acorbi Agreed not an important issue. Just wanted to log in case it does become an issue in future. There were a few records that had this issue.
